python/py-userpath/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-userpath
-version             1.4.1
+version             1.6.0
 categories-append   devel
 license             {MIT Apache-2}
 maintainers         {@0az 0az@afzhou.com} openmaintainer
```

Comment:
not your fault, but can you take care of the `port lint` warning: 
`Warning: Maintainer email address should be obfuscated as afzhou.com:0az` as well?

Reply:
Done.